### PR TITLE
More crimes for the jailbird trait

### PR DESCRIPTION
**Pull request by Bartimeus973**
## About the PR 
Short and sweet PR, adding some simple minor and major crime entries to the possible ones being picked when someone has the jailbird trait, ranging from humorous to concerning.

## Why's this needed? 
More variety is good! Also shakes things up a bit since this hasn't been changed in a good while.

## Changelog

```changelog
(u)Bartimeus973
(+)Jailbirds are now wanted for a bunch of new crimes!
```

### CODE REVIEW

File: code/datums/datacore.dm [hunk 1]
```diff
@@ -123,7 +123,27 @@
...
+								"Disregarding previous tickets.",\
+								"Fashion crimes.",\
+								"Gambling.",\
+								"Bribery.",\
+								"Sleeping on the job.",\
+								"Unauthorized stamp collecting.",\
+								"Refusing to wash their hands.",\
+								"Maintenance lurking.",\
+								"Dumpster diving.",\
+								"Not putting their hands in front of their mouth when sneezing.",\
```
Comment (flrsh) [inline code/datums/datacore.dm:137]: Nit but maybe reword as "not covering their mouth when sneezing"?

**Review by TTerc — APPROVED** [review]
